style.css

```diff
@@ -2,7 +2,7 @@
 @import url("/components/cart/cart.css");
 @import url("/components/wishlist/wishlist.css");
 @import url("/components/product-list/product-list.css");
-@import url("/components/login/login.css");
+@import url("/components/auth/auth.css");
 
 .main-container-width {
   max-width: 1200px;
```

Comment:
Try to use relative units instead of absolute units.

Reply:
It is main container width so I've tried to give it an intrinsic value